core/templates/dev/head/components/side_navigation_bar/side_navigation_bar_directive.html

```diff
@@ -77,6 +77,9 @@
       height: 56px;
       padding-top: 6px;
     }
+    .oppia-sidebar-menu-icon.fa {
```

Comment:
Oh, I see. Was assuming that whole class has to do with margin or placement. Looks like it isn't. Thanks! Pushed another commit